largestProduct.js

```diff
@@ -1,6 +1,25 @@
 class LargestProduct {
   constructor(number) {
+    this.number = number;
+    this.index = 0;
+  }
+  getLargestProduct() {
+    let temp = 0;
+    let arr = this.number.split('');
+    for (let i = 0; i < arr.length; i++) {
+      let x = arr[i] * arr[i + 1] * arr[i + 2] * arr[i + 3] * arr[i + 4];
+      let y = arr[i + 1] * arr[i + 2] * arr[i + 3] * arr[i + 4] * arr[i + 5];
```

Comment:
I'm not sure if the y variable really does anything here since you're still going to calculate this value in the next iteration of the for loop